.github/workflows/main.yml

```diff
@@ -2632,9 +2623,9 @@ jobs:
 
       - name: Setup GraalVM {{ matrix.graal }}
         run: |
-          URL=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${{ matrix.graal }}/graalvm-ce-java11-linux-amd64-${{ matrix.graal }}.tar.gz
+          URL=https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${{ matrix.graal }}/graalvm-ce-java17-linux-amd64-${{ matrix.graal }}.tar.gz
           curl -L $URL | tar -xz
-          GRAALVM=`pwd`/graalvm-ce-java11-${{ matrix.graal }}
+          GRAALVM=`pwd`/graalvm-ce-java17-${{ matrix.graal }}
```

Comment:
for future reference: the test setup is still on an EOL release due to `gu` no longer supporting language extensions.

CI is using [`22.3.1`](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-22.3.1) which is the last known release which supports the full set of extensions:
```
graalvm-ce-java17-22.3.1]$ bin/gu available
Downloading: Component catalog from www.graalvm.org
ComponentId              Version             Component name                Stability                     Origin 
---------------------------------------------------------------------------------------------------------------------------------
espresso                 22.3.1              Java on Truffle               Supported                     github.com
espresso-llvm            22.3.1              Java on Truffle LLVM Java librSupported                     github.com
js                       22.3.1              Graal.js                      Supported                     github.com
llvm                     22.3.1              LLVM Runtime Core             Supported                     github.com
llvm-toolchain           22.3.1              LLVM.org toolchain            Supported                     github.com
native-image             22.3.1              Native Image                  Early adopter                 github.com
native-image-llvm-backend22.3.1              Native Image LLVM Backend     Early adopter (experimental)  github.com
nodejs                   22.3.1              Graal.nodejs                  Supported                     github.com
python                   22.3.1              GraalVM Python                Experimental                  github.com
R                        22.3.1              FastR                         Experimental                  github.com
ruby                     22.3.1              TruffleRuby                   Experimental                  github.com
visualvm                 22.3.1              VisualVM                      Experimental                  github.com
wasm                     22.3.1              GraalWasm                     Experimental                  github.com
```

I think this would be the [current](https://github.com/graalvm/graalvm-ce-builds/releases/tag/jdk-17.0.9) GraalVM 17 release:
```
graalvm-community-openjdk-17.0.9+9.1]$ bin/gu available
Downloading: Component catalog from www.graalvm.org
ComponentId              Version             Component name                Stability                     Origin 
---------------------------------------------------------------------------------------------------------------------------------
espresso                 23.0.2              Java on Truffle               Supported                     github.com
espresso-llvm            23.0.2              Java on Truffle LLVM Java librSupported                     github.com
icu4j                    23.0.2              ICU4J                         Supported                     github.com
js                       23.0.2              Graal.js                      Supported                     github.com
llvm                     23.0.2              LLVM Runtime Core             Supported                     github.com
llvm-toolchain           23.0.2              LLVM.org toolchain            Supported                     github.com
nodejs                   23.0.2.1            Graal.nodejs                  Supported                     github.com
regex                    23.0.2              TRegex                        Supported                     github.com
visualvm                 23.0.2              VisualVM                      Supported                     github.com
```

GraalVM releases based on JDK 21 or 22 don't have `gu` anymore.

Reply:
Yes .. if we want to have language integrations, we need to invent something for G21/22.